rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/reporting/freemarker/RenderLinkDirective.java

```diff
@@ -232,7 +232,7 @@ private void renderLink(Writer writer, String cssClass, ProjectModel project, St
         if (cssClass != null)
             writer.append(" class='" + cssClass + "'");
         writer.append(" href='").append(href);
-        appendProject(writer, project);
+//            appendProject(writer, project);
```

Comment:
this doesn't seem to be used anywhere on reports and breaks the HardcodedIP report links to source report of the file occurence

Reply:
(this is a followup on the standup discussion this morning, just to document the issue)

This is used in the following places:
https://github.com/windup/windup/blob/master/reporting/impl/src/main/resources/reports/templates/source.ftl#L216

https://github.com/windup/windup/blob/master/reporting/impl/src/main/resources/reports/templates/source.ftl#L24

It works around the fact that a single source file may be used in multiple projects. So depending upon which application was linked from, the source report may have to render a different navigation bar (because otherwise, the navigation bar links would not go to the right place).